lava_test_plans/testcases/master/template-android-xts.yaml.jinja2

```diff
@@ -0,0 +1,42 @@
+{% extends "testcases/master/template-master.jinja2" %}
+
+{## This template is not for generic use, please consider ##}
+{## to use the template-android-cts.yaml.jinja2 and ## }
+{## template-android-vts.yaml.jinja2 as possible as you can##}
```

Comment:
's| as possible as you can|.|g'

Reply:
it's not completely forbidden to use this template, it's just recommended to use the cts and vts templates.
so "as possible as you can" added here.